cipher_suite_go114.go

```diff
@@ -0,0 +1,35 @@
+// +build go1.14
+
+package dtls
+
+import (
+	"crypto/tls"
+)
+
+// Convert from our cipherSuite interface to a tls.CipherSuite struct
+func toTLSCipherSuite(c cipherSuite) *tls.CipherSuite {
+	return &tls.CipherSuite{
+		ID:                uint16(c.ID()),
+		Name:              c.String(),
+		SupportedVersions: []uint16{0xfefd},
```

Comment:
Would be nice to have public const like https://github.com/golang/go/blob/a52821569310ca3807f74993af4019dfcaa74cfe/src/crypto/tls/common.go#L29-L38.

Reply:
So we have a `protocolVersion`, but it's a struct with `dtlsMajor` and `dtlsMinor`. Also unexported.

I suppose we could add a `VersionDTLS12` to match stdlib?